src/mpl/configure.ac

```diff
@@ -1247,6 +1247,14 @@ PAC_ARG_SHARED_MEMORY
 
 dnl always check POSIX shared memory
 AC_CHECK_FUNC(shm_open)
+if test "$ac_cv_func_shm_open" != yes ; then
+    # We may need -lrt to link shm_open. This is true on older linux.
+    AC_CHECK_LIB([rt],[shm_open],[have_rt=yes])
+    if test "$have_rt" = "yes" ; then
+        PAC_PREPEND_FLAG([-lrt],[LIBS])
+        ac_cv_func_shm_open=yes
+    fi
+fi
```

Comment:
should be cleaner to use `AC_SEARCH_LIBS([shm_open], [rt], [have_shm_open=yes])`?

Reply:
Good idea!